ROS 2 rosdistro db has been merged into ros/rosdistro.

nuclearsandwich opened the pull request:
See https://github.com/ros2/rosdistro/issues/492

I believe this is sufficient for the next rosindex job to run correctly.
One concern I had was whether merging the index files would cause all
distros to be counted double, as both ROS 1 and ROS 2 distros, but since
the working list of individual distributions is stored in _config.yml
that shouldn't be a problem. All the same I'd be glad if someone from
the rosindex core maintainers could check my work.

nuclearsandwich committed:
ROS 2 rosdistro db has been merged into ros/rosdistro.

See https://github.com/ros2/rosdistro/issues/492

I believe this is sufficient for the next rosindex job to run correctly.
One concern I had was whether merging the index files would cause all
distros to be counted double, as both ROS 1 and ROS 2 distros, but since
the working list of individual distributions is stored in _config.yml
that shouldn't be a problem. All the same I'd be glad if someone from
the rosindex core maintainers could check my work.

dirk-thomas commented:
Could we instead remove the `ros2_rosdistro` entry from the `_data/remotes.yml` file as well as the `_remotes/ros2_rosdistro` entry from the `index.yml`?

nuclearsandwich commented:
> Could we instead remove the `ros2_rosdistro` entry from the `_data/remotes.yml` file as well as the `_remotes/ros2_rosdistro` entry from the `index.yml`?

I didn't think so at first but reading more of the internals yes I believe that we can.

nuclearsandwich committed:
Remove separately fetched ros2_rosdistro.

It has merged with the canonical ROS rosdistro index and there is no
need to fetch it twice.

hidmic reviewed (approved):
LGTM
